src/select-menu/src/SelectMenuContent.js

```diff
@@ -45,9 +49,11 @@ export default class SelectMenuContent extends PureComponent {
       height,
       options,
       hasTitle,
+      hasTooltip,
       hasFilter,
       close,
       title,
+      tooltipContent,
```

Comment:
Could you rename this prop to `tooltip`, so that it matches `hasTooltip` prop naming?